bin/collator/src/local/chain_spec.rs

```diff
@@ -55,6 +55,11 @@ pub fn development_config() -> ChainSpec {
                     get_account_id_from_seed::<sr25519::Public>("Charlie"),
                     get_account_id_from_seed::<sr25519::Public>("Eve"),
                     get_account_id_from_seed::<sr25519::Public>("Ferdie"),
+                    // Arrakis.TEST account in MetaMask
+                    // Use known Arrakis.TEST account with private key
+                    // 0x...
+                    AccountId::from_ss58check("5Gbgok6b2HRpPDm7vSEDffBDsLNmAiFEgRSZcLePVYqKqT3i")
+                        .unwrap(),
```

Comment:
this is the H160 account which you can add to the Metamask and it will have preloaded tokens. Same as Alice but this is on H160.
I forgot to add the test key, will do it now. Feel free to name it however you want. I recently watched Dune and named the account  after the planet 🪐